skrub/_data_ops/tests/test_data_ops.py

```diff
@@ -363,7 +363,7 @@ def test_apply_bad_params(why_no_wrap, bad_param):
     else:
         cols = s.all()
     how = "cols" if bad_param == "how" else how
-    allow_reject = True if bad_param == "allow_reject" else False
+    allow_reject = bad_param == "allow_reject"
```

Comment:
the current version is less readable 
```suggestion
    allow_reject = True if bad_param == "allow_reject" else False
```